test/test_demo_data.py

```diff
@@ -21,3 +21,6 @@ def test_bmt_main(demo_data_obj):
 
 def test_shap(demo_data_obj):
     return demo_data_obj.run_shap()
+
+
+DemoData().run_vae_main()
```

Comment:
Can we remove this line? The reason is that we only want to collect the tests in the scripts and `pytest` will then automatically summarize all test functions whose names start with `test_`.